Building-with-IntelliJ-IDEA.md

```diff
@@ -20,104 +21,138 @@ You can build RuneLite locally using [JDK 11](https://adoptium.net/temurin/relea
 
-If you are having issues with missing JDK see [Troubleshooting](#troubleshooting) section.
-You can now save this build step as *Run configuration*, just right click *Install* again and select *Create..* (*Modify Run Configuration...* in IntelliJ 2021.2), window like this should pop-up, so just hit *Ok* (you can append `-DskipTests` for faster builds):
+![install_skip_tests](img%2Fbuilding-with-intellij%2Finstall-skip-tests.png)
 
-![project-maven-install-save](https://i.imgur.com/vdJoJ7L.png)
+Press ``OK``. You can now easily run your run configurations in the Maven window by right-clicking ``RuneLite`` (parent) > ``Run Configurations`` > ``runelite-parent [install] skip tests`` and selecting ``Run``.  
+Alternatively, you can select your run configuration in the dropdown box at the top and hit the ``Run`` button.
 
-Now, we need to tell IntelliJ to pick up changes based on Maven build (it should do that automatically, but sometimes it doesn't). So, still in same Maven window, just click the Refresh icon:
+![custom_configuration_top](img%2Fbuilding-with-intellij%2Fcustom-configuration-top.png)
 
-![project-maven-refresh](https://i.imgur.com/L2SmWtF.png)
+Other useful run configurations include:  
+``clean install -DskipTests -U``  which is your first troubleshooting step that you should run multiple times in case of a build failures  
+``-Dcheckstyle.skip=false checkstyle:check`` to only run checkstyle without building the client  
+``surefire:test`` to only run the tests without building the client  
+
+We now need to tell IntelliJ to pick up changes based on Maven build (it should do that automatically, but sometimes it doesn't). Click on `Generate Sources and Update folders for all projects` in the Maven window.
+
+![generate_sources.png](img%2Fbuilding-with-intellij%2Fgenerate_sources.png)
 
 ## Running the project
 
-Now locate *Project* in sidebar and click that open project view, and expand the tree to `runelite/runelite-client/src/main/java/net/runelite/client`. And all you need to do now is right-click `RuneLite` class and select run, and you are done, RuneLite should open:
+Now locate `Project` in the sidebar, and in the project view expand the tree to `runelite/runelite-client/src/main/java/net/runelite/client`. 
+And all you need to do now is right-click the `RuneLite` class and select `Run ‘RuneLite.main()’`
+
+![run_runelite.png](img%2Fbuilding-with-intellij%2Frun_runelite.png)
+
+If you receive a notification regarding Lombok's annotation processing, press ``Enable annotation processing``:
 
-![project-view-run-main](https://i.imgur.com/RXkMc48.png)
+![lombok_annotation_processing](img%2Fbuilding-with-intellij%2Flombok-annotation-processing.png)
 
-If you are having any issues with this step see [Troubleshooting](#troubleshooting) section.
+After enabling annotation processing you will need to rebuild Maven as mentioned in [Building the project](#building-the-project), and then run `Runelite.main()` again.
```

Comment:
```suggestion
```

This is not really true. Enabling annotation processing and having to rebuild are unrelated. Sometimes maven likes to be maven and you have to just more maven it up. It doesn't always happen, but sometimes it does.
This addition can be removed since it's already covered in multiple parts:
- Two lines below, it notes that you should refer to the troubleshooting section in case you got any problems. The troubleshooting steps have the ``Client failing to start`` note
- We mention clean install -DskipTests -U in the custom run configuration part already

I'm fine with clarifying it in the troubleshooting section as gamma mentioned

Reply:
suggestion applied here: 329a1273f9555758211c5fa79f9a52bd90d91b00

clarification hasn't been done